### PR TITLE
timestamp support for kafka events

### PR DESCRIPTION
**Pull request by mariam-japaridze**
I have added timestamp support for 0.11.x version, please check it out and tell me if I need to change something. I have tested for the 0.11.x version and it worked

### CODE REVIEW

**Comment by mariam-japaridze** [comment]
#141 issue

File: kafka_consumer.c [hunk 1]
```diff
@@ -434,6 +435,12 @@ static void consumer_commit(int async, INTERNAL_FUNCTION_PARAMETERS) /* {{{ */
                 return;
             }
 
+            ztimestamp = rdkafka_read_property(NULL, zarg, ZEND_STRL("timestamp"), 0 TSRMLS_CC);
+            if (!ztimestamp) {
+                zend_throw_exception(ce_kafka_exception, "Invalid argument: Specified Message has no timestamp", RD_KAFKA_RESP_ERR__INVALID_ARG TSRMLS_CC);
+                return;
+            }
```
Comment (arnaud-lb) [inline kafka_consumer.c]: Is it invalid to call rd_kafka_commit() when a message has no timestamp ?

**Review by arnaud-lb — COMMENTED** [review]
Thank you for your contribution. I've reviewed the changes, and I'm wondering if the exception in consumer_commit() is necessary ?

File: kafka_consumer.c (hunk 1 above)
Comment (mariam-japaridze) [reply kafka_consumer.c]: I'll fix that. I think it is not right to have the exception.

**Comment by mariam-japaridze** [comment]
The only issue I am trying to solve now, is the version before timestamp type was introduced in librdkafka. (<0.9). Any ideas how to solve this in C?

**Comment by arnaud-lb** [comment]
Thanks, merged :) I took care of supporting 0.9: https://github.com/arnaud-lb/php-rdkafka/commit/58e93510d04ce9c85ddb685f4d7af13f3cae7137

**Pull request merged**